### PR TITLE
DOC: Release Notes Sept 16, 2015

### PR DESCRIPTION
**Pull request by catong**
https://openedx.atlassian.net/browse/DOC-2255
Release Notes for Sept 16, 2015 release.
### Reviewers

Check the appropriate box after each reviewer finishes and gives :+1:. 
- [x] Doc team review: @lamagnifica 
- [x] Product/SME review: @explorerleslie 
- [x] Product/SME review: @pbaruah 
- [x] Product/SME review: @clrux 
- [x] Product/SME review: @sarina 
### Post-review
- [  ] Squash commits

### CODE REVIEW

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+*************
+Accessibility 
+*************
+
+Several accessbility-related changes were made in the LMS Dashboard and
+courseware templates, including screen reader navigation improvements and
+explicitly defined hover, link, and foreground/background colors for adequate
+contrast in screen reader elements. (AC-155, AC-166)
```
Comment (downzer0) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: You can remove AC-166 as it's extremely minor and possibly confusing.

**Comment by explorerleslie** [comment]
:+1: from TNL perspective

File: en_us/release_notes/source/2015/openedx/openedx_0916_2015.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+Developers who are interested in extending Open edX Insights can now set up a
+separate development environment, the Open edX Analytics Developer stack, to
+support analytics development. This environment, known as the Analytics
+Devstack, includes the edX Analytics Data API and Open edX Insights, as well
+as all of the components needed to run the Open edX Analytics Pipeline.
+
+For more information, see `Installing the Open edX Analytics Developer Stack`_
+in the *Installing, Configuring, and Running the Open edX Platform* guide.
+
```
Comment (catong) [inline en_us/release_notes/source/2015/openedx/openedx_0916_2015.rst]: @sarina ^

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  the re-verification deadline. This issue has been resolved. Learners can
+  check whether they satisfied the verification/credit requirement on their
+  **Progress** pages. A green checkmark appears next to the verification
+  requirement when photos are approved by the verification service. The
+  verification might take several days. (ECOM-2225)
+
+* In for-credit courses, credit-eligible assessments were listed on the
+  **Progress** page in the order that they were created. They are now sorted by
+  assessment start date. (ECOM-2118)
+
```
Comment (catong) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst:38]: @pbaruah ^

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: maybe precede these with a heading like "Corrected Errors" or "Bug Fixes"? I thought they were all going to be about new features, so the past tense surprised me...

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  assessment start date. (ECOM-2118)
+
+* When learners select **Show Answer** for dropdown problems, the positioning of
+  the correct answer has been fixed so that it now displays in the same way as
+  it does for other problem types such as numerical input. (OSPR-794)
+
+* When learners select **Show Answer** for math expression input problems,
+  the styling has been modified to improve the display of answers as well as
+  of the loading animation when **Show Answer** is selected. (OSPR-808)
+
```
Comment (catong) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: @sarina ^

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+* In courses containing in-course reverification steps, learners on the
+  verified track who submitted photo IDs for an in-course reverification
+  checkpoint were incorrectly shown a message indicating that they had missed
+  the re-verification deadline. This issue has been resolved. Learners can
+  check whether they satisfied the verification/credit requirement on their
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: can ^now check (?)
verification ^or credit (?)

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  **Progress** pages. A green checkmark appears next to the verification
+  requirement when photos are approved by the verification service. The
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: when photos --> after submitted photos (?)

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+  verification might take several days. (ECOM-2225)
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: verification might  --> verification process can (?)

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+* In for-credit courses, credit-eligible assessments were listed on the
+  **Progress** page in the order that they were created. They are now sorted by
+  assessment start date. (ECOM-2118)
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: what do you mean by the start date here? the subsection release date?

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+* When learners select **Show Answer** for dropdown problems, the positioning of
+  the correct answer has been fixed so that it now displays in the same way as
+  it does for other problem types such as numerical input. (OSPR-794)
+
+* When learners select **Show Answer** for math expression input problems,
+  the styling has been modified to improve the display of answers as well as
+  of the loading animation when **Show Answer** is selected. (OSPR-808)
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: maybe,

When learners select **Show Answer** for math expression input problems,
improvements have been made to the answer loading animation and to the final answer display. (?)

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst (hunk 1 above)
Comment (downzer0) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: As for the wording, how about something like this:

"Several accessibility-related improvements were made in the LMS, Dashboard, and courseware templates that aid readability and navigability."

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst (hunk 5 above)
Comment (sarina) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: Yes, those sound right. I'm a little surprised to see them in the release notes - they seem like fairly minor ux bugfixes to me.

File: en_us/release_notes/source/2015/openedx/openedx_0916_2015.rst (hunk 2 above)
Comment (sarina) [inline en_us/release_notes/source/2015/openedx/openedx_0916_2015.rst]: :+1:

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst (hunk 9 above)
Comment (catong) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: I'm actually waiting on a clarification of what date is used for sorting; might be the assessment due date.

**Comment by lamagnifica** [comment]
OK, Carol, maybe for the LMS section:

(you can just click edit and copy and paste the marked up rst text)
Beth has reviewed.

# 

# Introducing edX as an LTI Tool Provider

In a pilot implementation, the edX Edge site is now being used as a learning
tool interoperability (LTI) tool provider. Course teams use this feature to
present content from edX courses, including advanced problem types and videos,
in their on-campus learning management systems. The edX as an LTI Tool Provider
feature can make it easier for you to teach with, and students to learn from,
the best materials available, regardless of platform.

.. note:: Support for this feature is currently provisional. EdX is working 
 with our early adopter partners to support further testing of this feature
 with additional learning management systems.

For more information, see `Using edX as an LTI Tool Provider`_ in _Building and
and Running an edX Course_.

.. _Using edX as an LTI Tool Provider: http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/building_course/lti/index.html

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst (hunk 5 above)
Comment (catong) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: @sarina They are fairly minor, but I thought they were good improvements to list + good to include open source contributions.

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+in their on-campus learning management systems. The "edX as an LTI Tool
+Provider" feature can make it easier for you to teach with, and students to
+learn from, the best materials available, regardless of platform.
+
+.. note:: Support for this feature is currently provisional. EdX is working 
+ with our early adopter partners to support further testing of this feature
+ with additional learning management systems.
+
+For more information, see `Using edX as an LTI Tool Provider`_ in the *Building and
+and Running an edX Course* guide.
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: my bad -- two "and"s

**Comment by lamagnifica** [comment]
one typo then :+1:

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+=========
+
+* In courses containing in-course reverification steps, learners on the
+  verified track who submitted photo IDs for an in-course reverification
+  checkpoint were incorrectly shown a message indicating that they had missed
+  the re-verification deadline. This issue has been resolved. Learners can
+  continue to check whether they satisfied the verification or credit
+  requirement on their **Progress** pages. A green checkmark appears next to the
+  verification requirement when submitted photos are approved by the
+  verification service. The verification process can take several days.
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst]: can take several days --> can take 24 hours or more

File: en_us/release_notes/source/2015/lms/lms_0916_2015.rst [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+  (ECOM-2225)
+
+* In for-credit courses, credit-eligible assessments were listed on the
+  **Progress** page in the order that they were created. They are now sorted by
+  assessment due date. (ECOM-2118)
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0916_2015.rst:38]: thumbs from @pbaruah  otherwise on both of these

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/556

**Pull request merged**